src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -10,13 +10,27 @@ import {
 import { DetailsTabAlert } from "./components/DetailsTabAlert";
 import { format } from "date-fns";
 
+import { InstanceType } from "../../utils";
+
 type KafkaDetailsTabProps = {
   id: string;
   createdAt: Date;
   updatedAt: Date;
   expiryDate?: Date;
   owner: string;
   region: string;
+  instanceType: InstanceType;
+  size?: number;
```

Comment:
Should this be optional? None of the other options are.

Reply:
`size` should not display for trial instances, and `expiryDate` should not display for standard instances. Those should be the only properties that are different between trial and standard instances.

Reply:
Ah, I didn't know that and it wasn't obvious looking at just the code. I'd make two separate components then, one for trial instances and one for standard instances. This way it's obvious what it is the required data for both cases.